tests/integration_tests/test_public_api.py

```diff
@@ -137,3 +137,22 @@ def test_public_api(cloud: BaseCloud):
         "Unable to find daily development image for "
         f"{cloud._type}:{latest_devel_release}"
     )
+    print(f"Checking latest minimal daily devel image: {latest_devel_release}")
+
+    if isinstance(cloud, pycloudlib.LXDContainer):
+        print(f"Checking latest daily minimal image: {latest_devel_release}")
+        released_minimal_image_id = cloud.daily_image(
+            release=latest_lts, image_type=ImageType.MINIMAL
+        )
+        with cloud.launch(image_id=released_minimal_image_id) as instance:
+            instance.wait()
+            assert (
+                "status: done" == instance.execute("cloud-init status").stdout
+            )
+            assert (
+                "minimal"
+                in instance.execute(
+                    "grep build_name /etc/cloud/build.info"
+                ).stdout
+            )
+            instance.delete()
```

Comment:
should this test be added to an lxd integration test instead? given that this will only run on lxd and the public api is unaffected by this change?

Reply:
Good suggestion, I pulled it into a separate test and also parametrized that test to cover GCE and Ec2 which also support ImageType.MINIMAL image_type overrides in released_image/daily_image